docs/v0.9/en/getting-started/installation/greptimedb-cluster.md

```diff
@@ -1,73 +1,150 @@
 # GreptimeDB Cluster
 
-The GreptimeDB cluster can run in the [cluster](/contributor-guide/overview.md) mode to scale horizontally.
+The GreptimeDB cluster can run in cluster mode to scale horizontally. For production usage, we highly recommend deploying it in Kubernetes.
 
-## Install gtctl and run playground
+## Use Docker Compose
 
-[gtctl](https://github.com/GreptimeTeam/gtctl) is the cli tool for managing the GreptimeDB cluster. You can use the following one-line installation(only for Linux and macOS):
+###  Prerequisites
+
+Using Docker Compose is the easiest way to run the GreptimeDB cluster. Before starting, make sure you have already installed the Docker.
+
+### Step1: Download the YAML file for Docker Compose
 
 ```
-curl -fsSL \
-  https://raw.githubusercontent.com/greptimeteam/gtctl/develop/hack/install.sh |\
-  sh
+wget https://raw.githubusercontent.com/GreptimeTeam/greptimedb/main/docker/docker-compose/cluster-with-etcd.yaml
 ```
 
-:::tip Note
+### Step2: Start the cluster
 
-For Windows users, due to the complexity and compatibility of running multiple components together, we strongly recommend enabling WSL([Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/about)) on your operating system, and lunch a latest Ubuntu to proceed.
+```
+GREPTIMEDB_VERSION=<%greptimedb-version%> \
+  docker compose -f ./docker/docker-compose/cluster-with-etcd.yaml up 
+```
 
-:::
+If the cluster starts successfully, it will listen on 4000-4003. , you can access the cluster by referencing the [Quick Start](../quick-start.md).
 
-Once the download is completed, the binary file `gtctl` will be stored in your current directory.
+### Clean up
 
-The **fastest** way to experience the GreptimeDB cluster is to use the playground:
+You can use the following command to stop the cluster:
 
 ```
-./gtctl playground
+docker compose -f ./cluster-with-etcd.yaml down
 ```
 
-When the command is executed, the playground will be started in the foreground. You can use `Ctrl+C` to stop the playground. The playground will deploy the minimal GreptimeDB cluster in bare-metal mode on your host.
+By default, the data will be stored in `/tmp/greptimedb-cluster-docker-compose`. You also can remove the data directory if you want to clean up the data:
 
-If everything is ok, you can visit the Dashboard via `http://localhost:4000/dashboard/`.
-
-For more details, please refer to [gtctl operations](/reference/gtctl.md).
+```
+rm -rf /tmp/greptimedb-cluster-docker-compose
+```
```

Comment:
Instead of teaching user how to cleanup the data. I suggest to teach user how to customize this path. We can leave a sentence say if you remove the path you get a fresh install. But don't show `rm -rf ...` here. I feel it's something negative..

Reply:
Sadly, I forget to make the data path configurable in this version. Maybe I can use the following description by removing the `rm -rf`:

```
...
By default, the data will be stored in `/tmp/greptimedb-cluster-docker-compose`. You also can remove the data directory if you want to clean up the data.
...
```

Reply:
Yes, we can